test/e2e/specs/editor/various/block-editor-keyboard-shortcuts.spec.js

```diff
@@ -0,0 +1,220 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+async function addTestParagraphBlocks( { editor, page } ) {
```

Comment:
Nit: What do you think if we just use `editor.setContent` or `createNewPost` with the `content` option? If it's easier to manage focus this way then nvm.

Reply:
I forgot about the `editor.setContent` 😅 I think it should work; IIRC, the `resetBlocks` should focus on the last block, where these tests expect it.